fdbserver/BlobWorker.actor.cpp

```diff
@@ -3081,6 +3123,57 @@ ACTOR Future<Void> runGRVChecks(Reference<BlobWorkerData> bwData) {
 	}
 }
 
+// FIXME: better way to do this?
+// monitor system keyspace for new tenants
+ACTOR Future<Void> monitorTenants(Reference<BlobWorkerData> bwData) {
+	loop {
+		state Reference<ReadYourWritesTransaction> tr = makeReference<ReadYourWritesTransaction>(bwData->db);
+		loop {
+			try {
+				tr->setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
+				tr->setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
+				state RangeResult tenantResults;
+				wait(store(tenantResults, tr->getRange(tenantMapKeys, CLIENT_KNOBS->TOO_MANY)));
```

Comment:
I intend to clarify this soon, but it's possible we'll need to support more than `TOO_MANY` tenants. Just something to keep in mind.